src/ru/acomics/src/eu/kanade/tachiyomi/extension/ru/acomics/AComics.kt

```diff
@@ -39,17 +40,32 @@ class AComics : ParsedHttpSource() {
     override fun popularMangaRequest(page: Int): Request =
         GET("$baseUrl/comics?$DEFAULT_COMIC_QUERIES&sort=subscr_count&skip=${10 * (page - 1)}", headers)
 
-    override fun popularMangaSelector() = "table.list-loadable > tbody > tr"
+    override fun popularMangaSelector() = "section.serial-card"
 
     override fun popularMangaFromElement(element: Element) = SManga.create().apply {
-        thumbnail_url = element.selectFirst("a > img")?.absUrl("src")
-        element.selectFirst("div.title > a")!!.run {
+        thumbnail_url = element.selectFirst("a > img")?.absUrl("data-real-src")
+        element.selectFirst("h2 > a")!!.run {
             setUrlWithoutDomain(attr("href") + "/about")
             title = text()
         }
     }
 
-    override fun popularMangaNextPageSelector() = "span.button:not(:has(a)) + span.button > a"
+    override fun popularMangaNextPageSelector(): String? {
+        TODO("Not yet implemented")
```

Comment:
You are looking for `a.infinite-scroll` after the last `div.infinite-scroll-content > section.serial-card`. That element only exists if there is more content to load. You can make it as advanced or basic as you want.